.dockerignore

```diff
@@ -11,3 +11,4 @@ compile_commands.json
 nix/
 !README.md
 !VERSIONING.md
+cmake-build-*/
```

Comment:
This was probably committed on accident.

Reply:
I'd be fine with leaving this in tbh, it's not harmful at all.